images/plbase/python-requirements.txt

```diff
@@ -14,20 +14,20 @@ networkx==3.2.1
 nltk==3.8.1
 numpy==1.26.4
 openpyxl==3.1.2
-pandas-stubs==2.2.0.240218
+pandas-stubs==2.2.1.240316
 pandas==2.2.1
 Pint==0.23
 psutil==5.9.8
 PuLP==2.8.0
 pycryptodome==3.20.0
 Pygments==2.17.2
-pyarrow==15.0.0
+pyarrow==15.0.2
 pygments-ansi-color==0.3.0
 pygraphviz==1.12
 pyquaternion==0.9.9
-pytest-cov==4.1.0
+pytest-cov==5.0.0
```

Comment:
The release notes for 5.0.0 are pretty light (https://pytest-cov.readthedocs.io/en/latest/changelog.html), but if CI runs I think the probability of breaking something is really low.